### PR TITLE
Chain transform helper

### PR DESCRIPTION
**Pull request by st--**
To simplify working with positive rescales...

### CODE REVIEW

File: gpflow/transforms.py [hunk 1]
```diff
@@ -169,52 +204,54 @@ def log_jacobian_tensor(self, x):
...
     """
-    A transform that can linearly rescale parameters, in conjucntion with
-    another transform. By default, the identity transform is wrapped so
+    A transform that can linearly rescale parameters:
     .. math::
        y = factor * x
 
-    If another transform t() is passed to the constructor, then this transform becomes
-    .. math::
+    Use `Chain` to combine this with another transform such as Log1pe:
```
Comment (jameshensman) [inline gpflow/transforms.py]: Great docstring.

File: gpflow/core/base.py [hunk 2]
```diff
@@ -44,8 +44,6 @@ def __str__(self):
 
 class ITransform(metaclass=abc.ABCMeta):
     """
-    The terminology around Transforms can be a bit confusing.
-
     x is the unconstrained, free-space parameter, which can take any value.
```
Comment (awav) [inline gpflow/core/base.py]: @st-- In fact it is quite the contrary :)

File: tests/test_transforms.py [hunk 3]
```diff
@@ -191,10 +214,10 @@ def setUp(self):
     def test_forward_backward(self):
         free_1d = np.random.randn(8)
         fwd1d = self.t1.forward(free_1d)
-        assert_allclose(fwd1d.shape, np.array([len(free_1d), self.t1.dim, self.t1.dim]))
+        assert_allclose(fwd1d.shape, np.array([8, 1, 1]))
```
Comment (awav) [inline tests/test_transforms.py]: Why it was changed?

Comment (jameshensman) [reply tests/test_transforms.py]: reverting

**Pull request merged**